src/shared/components/palette/case-flag/read-case-flag-field.component.html

```diff
@@ -5,10 +5,20 @@
   <ng-container *ngSwitchDefault>
     <h2 class="govuk-heading-l">Case flags</h2>
     <div *ngFor="let flagData of partyLevelCaseFlagData">
-      <ccd-case-flag-table *ngIf="flagData.partyName" [flagData]="flagData" [firstColumnHeader]="'Party level flags'"></ccd-case-flag-table>
+      <ccd-case-flag-table *ngIf="flagData.partyName"
+        [tableCaption]="flagData.partyName"
+        [flagData]="flagData"
+        [firstColumnHeader]="'Party level flags'"
+      ></ccd-case-flag-table>
     </div>
     <div *ngIf="caseLevelCaseFlagData">
-      <ccd-case-flag-table [flagData]="caseLevelCaseFlagData" [firstColumnHeader]="'Case level flags'"></ccd-case-flag-table>
+      <div *ngIf="caseLevelCaseFlagData">
```

Comment:
@johnbenjamin-hmcts - looks like you have an extra `<div *ngIf="caseLevelCaseFlagData">`